src/applications/disability-benefits/2346/Components/VeteranInformationPage.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { getVeteranInformationData } from '../actions';
+
+class VeteranInformationPage extends React.Component {
+  componentDidMount() {
+    this.props.getVeteranInformationData();
+  }
+  render() {
+    const { first, last, gender, dateOfBirth } = this.props;
+    return (
+      <div>
+        <p>This is the personal information we have for you.</p>
+        <div>
+          <div className="usa-alert schemaform-sip-alert">
+            <div className="usa-alert-body">
+              {first} {last} <br />
```

Comment:
line breaks ```<br />``` are not really desirable in markup.  Can you use styling to achieve the same goal?

Reply:
https://design.va.gov/utilities/margins

Reply:
Done @kelsonic  and @bradl3yC thanks for the link to that resource.